src/providers/fault-count/fault-count.ts

```diff
@@ -110,10 +111,20 @@ export class FaultCountProvider {
     }
   }
 
-  public getManoeuvreFaultCount = (category: TestCategory, data: object, faultType: CompetencyOutcome): number => {
+  public getManoeuvreFaultCount = (
+    category: TestCategory,
+    data: object | Manoeuvres[],
```

Comment:
Can we be more specific here? you can basically pass in `any` like this

Reply:
Do you mean replace `object`?

Reply:
yeah, `object` can mean any object

Reply:
That's pre-existing code and consistent with CatB handling.  I'm happy to fix tech debt but I'd rather do it in a separate ticket and address all cases.

Reply:
I think this is the right time to fix this because there is not such story which would cover this and it'll be left as it is once the PR gets merged.

Also the comment below relies on this.